src/api/spec/controllers/webui/user_controller_spec.rb

```diff
@@ -0,0 +1,184 @@
+require 'rails_helper'
+
+RSpec.describe Webui::UserController do
+  let!(:user) { create(:confirmed_user, login: "tom") }
+  let!(:non_admin_user) { create(:confirmed_user, login: "moi") }
+  let!(:admin_user) { create(:admin_user, login: "king") }
+  let(:deleted_user) { create(:deleted_user) }
+
+  describe "GET #index" do
```

Comment:
This is already tested in [spec/controllers/webui/webui_controller_spec.rb](https://github.com/openSUSE/open-build-service/blob/master/src/api/spec/controllers/webui/webui_controller_spec.rb) you should only test that the before filter is used like in [spec/controllers/webui/configuration_controller_spec.rb](https://github.com/openSUSE/open-build-service/blob/master/src/api/spec/controllers/webui/configuration_controller_spec.rb#L11)